apps/dashboard/src/components/wallets/ConnectWalletPlayground/ThemeButton.tsx

```diff
@@ -11,22 +12,20 @@ export function ThemeButton(props: {
 }) {
   const trackEvent = useTrack();
   const bg = props.theme === "dark" ? "black" : "white";
-  const borderColor = props.theme === "dark" ? "gray.800" : "gray.200";
+  const borderColor =
+    props.theme === "dark" ? "border-gray-800" : "border-gray-200";
   return (
-    <Tooltip label={props.theme === "dark" ? "Dark" : "Light"}>
+    <ToolTipLabel label={props.theme === "dark" ? "Dark" : "Light"}>
       <Button
         disabled={props.disabled}
-        cursor={props.disabled ? "not-allowed" : "pointer"}
-        w={10}
-        h={10}
-        borderRadius="50%"
+        className={cn(
+          props.disabled ? "cursor-not-allowed" : "cursor-pointer",
+          "h-10 w-10 rounded-full border-[3px] border-border",
+          `bg-${bg}`,
+          `hover:bg-${bg}`,
```

Comment:
@kien-ngo please address this - you can't use string interpolation with tailwind - The tailwind styles are not generated at runtime - if the tailwind compiler sees `bg-black` it generates that style and adds it in CSS file - here it can't see that and the required style may never get added in the CSS bundle

Reply:
yeah you have mentioned this before too. I will update the code ty ser

Reply:
it works in local tho...

Reply:
It may appear to work if the required styles happen to be included in the page because of other sections